server/src/Game/Game.js

```diff
@@ -86,6 +86,12 @@ export default class Game {
         const dt = this._tick() / 1000
         Object.values(this.players).forEach(player => {
             player.socket.emit(EVENTS.SERVER_TICK, dt)
+            const pchange = player.update(dt)
```

Comment:
Could you rename `pchange` to _doesPlayerUpdate_ ? That makes it more readable
```suggestion
            const doesPlayerUpdate = player.update(dt)
```